django/middleware/common.py

```diff
@@ -53,7 +54,7 @@ def process_request(self, request):
                             'request': request
```

Comment:
I guess we should remove this logging call since `BaseHandler ` now also does logging.